vardbg/assets/styles/default.py

```diff
@@ -0,0 +1,47 @@
+from pygments.style import Style
```

Comment:
The assets directory is for static assets. There should not be any Python code in it.